solarforecastarbiter/reports/figures/plotly_figures.py

```diff
@@ -277,9 +286,11 @@ def timeseries(timeseries_value_df, timeseries_meta_df,
             name=_legend_text(metadata['forecast_name']),
             legendgroup=metadata['forecast_name'],
             marker=dict(color=next(palette)),
-            connectgaps=False),
+            connectgaps=False,
+            **plot_kwargs),
         )
         plotted_objects += 1
+    fig.update_traces(mode='lines+markers', marker=dict(size=6))
```

Comment:
comment out pending decision on whether or not its too busy

Reply:
yeah, it is too busy for most plots, but I like it for event plots. Up to you